template-parts/content-none.php

```diff
@@ -16,35 +16,36 @@
 
 	<div class="page-content">
 		<?php
-		if ( is_home() && current_user_can( 'publish_posts' ) ) : ?>
-
-			<p><?php
-				printf(
-					wp_kses(
-						/* translators: 1: link to WP admin new post page. */
-						__( 'Ready to publish your first post? <a href="%1$s">Get started here</a>.', '_s' ),
-						array(
-							'a' => array(
-								'href' => array(),
-							),
-						)
-					),
-					esc_url( admin_url( 'post-new.php' ) )
-				);
-			?></p>
-
-		<?php elseif ( is_search() ) : ?>
+		if ( is_home() && current_user_can( 'publish_posts' ) ) :
+
+			printf(
+				wp_kses(
+					/* translators: 1: link to WP admin new post page. */
+					'<p>' . __( 'Ready to publish your first post? <a href="%1$s">Get started here</a>.', '_s' ) . '</p>',
```

Comment:
As the `<p>` tags are fine and not a string with content we're not sure off, they should be outside of the `wp_kses()` call òr added to the whitelisted tags array.
 IMHO, outside of the call would be most logical.

`printf( '<p>' . wp_kses() . '</p>', $replacement );`